playwright-e2e/tests/dsm/mailing-list/rgp-mail-list-download.spec.ts

```diff
@@ -0,0 +1,120 @@
+import { expect } from '@playwright/test';
+import { login } from 'authentication/auth-dsm';
+import { APP } from 'data/constants';
+import { test } from 'fixtures/rgp-fixture';
+import { Miscellaneous } from 'lib/component/dsm/navigation/enums/miscellaneousNav.enum';
+import { Study } from 'lib/component/dsm/navigation/enums/selectStudyNav.enum';
+import { Navigation } from 'lib/component/dsm/navigation/navigation';
+import { SortOrder } from 'lib/component/table';
+import * as lodash from 'lodash';
+import MailingListPage, { COLUMN } from 'pages/dsm/mailing-list-page';
+import { WelcomePage } from 'pages/dsm/welcome-page';
+import TellUsYourStoryPage, { WHO } from 'pages/rgp/enrollment/tell-us-your-story-page';
+import HomePage from 'pages/rgp/home-page';
+import HowItWorksPage from 'pages/rgp/how-it-works-page';
+import { assertTableHeaders } from 'utils/assertion-helper';
+import { getDate, getMailingListDownloadedFileDate, mailingListCreatedDate } from 'utils/date-utils';
+import { generateEmailAlias } from 'utils/faker-utils';
+import { MailListCSV, readMailListCSVFile } from 'utils/file-utils';
+
+
+const RGP_USER_EMAIL = process.env.RGP_USER_EMAIL as string;
+const newEmail = generateEmailAlias(RGP_USER_EMAIL);
+
+test.describe.serial('When an interested participant does NOT meet participation requirements', () => {
+  test('Join Mail List @visual @dsm @rgp', async ({ page }) => {
+    const homePage = new HomePage(page);
+    await homePage.clickGetStarted();
+
+    const howItWorks = new HowItWorksPage(page);
+    await howItWorks.startApplication();
+
+    const tellUsYourStoryPage = new TellUsYourStoryPage(page);
+    await tellUsYourStoryPage.waitForReady();
+
+    // Check all but one checkbox (i.e. don't check all the boxes so that user fails to meet requirements)
+    await tellUsYourStoryPage.who().check(WHO.UnderstandEnglishOrSpanish);
+    await tellUsYourStoryPage.who().check(WHO.LivesInUS);
+    await tellUsYourStoryPage.who().check(WHO.HasRareGeneticallyUndiagnosedCondition);
+    await tellUsYourStoryPage.submit();
+
+    await expect(page.locator('text="Thank you for your interest in joining the Rare Genomes Project"')).toBeVisible();
+
+    const paragraphLocator = page.locator('.Paragraph');
+    await expect(paragraphLocator).toHaveCount(3);
+
+    expect(await paragraphLocator.nth(0).screenshot()).toMatchSnapshot('requirement-header-message.png');
```

Comment:
These screenshot comparisons failed for me locally. Am I missing something?
`Expected an image 770px by 83px, received 780px by 83px.`

Reply:
I don't have screenshots for window os. Is that case? If not, you probably are using a external monitor. It's okay for either case.